system/security/shared/seclib.hpp

```diff
@@ -19,6 +19,7 @@
 #define _SECLIB_HPP__
 
 #include "jlib.hpp"
+#include "jptree.hpp"
```

Comment:
Minor. Why did we need to add this? IPropertyTree was already part of this file so I presume it is already included

Reply:
Probably because I wanted to make my IDE stop complaining about a definition it couldn't seem to find through jlib.hpp, and I just forgot to remove it.